LayoutTests/fast/css/image-set-parsing.html

```diff
@@ -75,12 +75,12 @@
 testImageSetRule("Single -webkit-cross-fade for -webkit-mask-box-image-source",
                 "-webkit-mask-box-image-source",
                 "-webkit-cross-fade(url(\"#a\"), url(\"#b\"), 70%)",
-                "-webkit-cross-fade(url(\"#a\"), url(\"#b\"), 0.7) 1x");
+                "-webkit-cross-fade(url(\"#a\"), url(\"#b\"), 70%) 1x");
```

Comment:
Is this test change needed because of a behavior change outside of better `calc()` handling? Should we factor that behavior change out into a much smaller pull request that we can merge first?